### ISSUE TITLE
RuntimeError when I run the default run.sh on stage 5

### ISSUE THREAD

**Issue by happyjin** [espnet/espnet#2416]
I try to run default run.sh of HKUST Chinese dataset. During stage 5, there is RuntimeError in the code of `e2e_asr_transformer.py`. The code piece which trigger this error is as below:
```
local_best_scores, joint_best_ids = torch.topk(
   local_scores, beam, dim=1)
```
Here is the bug details:
```
Traceback (most recent call last):
  File "/home/jinli/projects/espnet/egs/hkust/asr1/../../../espnet/bin/asr_recog.py", line 327, in <module>
    main(sys.argv[1:])
  File "/home/jinli/projects/espnet/egs/hkust/asr1/../../../espnet/bin/asr_recog.py", line 305, in main
    recog(args)
  File "/home/jinli/projects/espnet/espnet/asr/pytorch_backend/asr.py", line 1034, in recog
    feat, args, train_args.char_list, rnnlm
  File "/home/jinli/projects/espnet/espnet/nets/pytorch_backend/e2e_asr_transformer.py", line 549, in recognize
    local_scores, beam, dim=1
RuntimeError: selected index k out of range
```
Then I print the `local_scores.shape` which returns `torch.Size([1, 6])`. However, the `beam=10` which means the shape of `local_scores` is less than `beam`. Could you guys please help me, how can I fix this bug? Thanks in advance!

**Comment by AndreyGalitsin** [comment]
I had the same problem.
I have reduced the **_beam-size_** parameter in **_decode_config_** file from 10 to 5, so it works for me.
Maybe it’s because the input sample is too short, so that the decoding result is even shorter than the specified beam size.

**Comment by happyjin** [comment]
@AndreyGalitsin Thanks for your reply. Did you also encounter this problem on **hkust dataset**? I got the incorrect result which gives me all zeros during decoding, you can find it as below:
![捕获](https://user-images.githubusercontent.com/18534928/92434523-9eed4f00-f1d2-11ea-875d-24e493b926fe.PNG)
How about your result after decoding? Do you know the possible reason why the result is incorrect?

**Comment by happyjin** [comment]
The problem is solved in #2433 . So I close this question.
